testutils/setup.go

```diff
@@ -278,6 +281,7 @@ func BuildTestSuiteConfig(testCfg TestConfig) config.AppConfig {
 				SvcManagerAddr:           svcManagerAddress,
 			},
 			MaxBlobSizeBytes: maxBlobLengthBytes,
+			PutRetries:       3,
```

Comment:
is the default behavior when `PutRetries=0` that it tries forever, which was causing tests to hang? We should probably change that behavior. IMO zero values (which are the default when forgotten) should always be safe (in this case should probably mean try once, or we insead just dissalow value 0). Unbounded behavior could for example be when `PutRetries=-1` instead.

Reply:
Will make the change in a follow up pr, see https://linear.app/eigenlabs/issue/DAINT-419/make-putretries-configuration-more-user-friendly